docker-compose.yml

```
@@ -262,7 +262,7 @@ services:
     ports:
       - "${ZIPKIN_PORT}:9411"
     healthcheck:
-      test: [ "CMD", "curl", "-f", "http://localhost:9411/health" ]
+      test: [ "CMD", "wget", "-q", "-O", "/dev/null", "http://localhost:9411/health" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)